### PR TITLE
Address issues flagged by golangci-lint 1.52

### PR DESCRIPTION
**Pull request by skitt**
See individual commits for details.

### CODE REVIEW

**Review by dfarrell07 — APPROVED** [review]

**Comment by tpantelis** [comment]
Go lint job failed:

```
pkg/workqueue/queue.go:78:3: directive `//nolint:revive // This is a while statement, we don't care about the empty block` is unused for linter "revive" (nolintlint)
		//nolint:revive // This is a while statement, we don't care about the empty block
```

This should succeed once https://github.com/submariner-io/shipyard/pull/1204 is merged but it's dependent on this PR. Rather than forcing merge this now, we can wait until #1204 is merged.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**
